modules/metering-upgrade-verify.adoc

```diff
@@ -0,0 +1,97 @@
+// Module included in the following assemblies:
+//
+// * metering/metering-upgrading-metering.adoc
+
+[id="metering-upgrade-verify_{context}"]
+= Verifying Metering Upgrade
+
+You can verify the metering upgrade by performing any of the following:
```

Comment:
As a user, I might want to see what my options are without having to scroll through all the nested bullets. So wondering if you could list out the 4 procedures with anchor links? I've thought of doing this myself but wasn't successful. 

> You can verify the metering upgrade by performing any of the following:
> - Verify the new metering version from the xref[web console] or xref[UI].
> - Verify that all required Pods are created using the xref[web console] or xref[UI].

But check my comment below first; with a little cleanup, you might be better avoiding anchors.